src/components/ParsedBotMessageBody.tsx

```diff
@@ -55,7 +55,7 @@ export default function ParsedBotMessageBody(props: Props) {
   // console.log('## sources: ', sources);
   if (tokens.length > 0) {
     return (
-      <MultipleTokenTypeContainer>
+      <MultipleTokenTypeContainer className="sendbird-word">
```

Comment:
It's a same className with UIKit. By adding this, user can customize the style of the message bubble. (It's another request from the customer)